asset-transfer-basic/application-go/assetTransfer.go

```diff
@@ -0,0 +1,146 @@
+/*
+Copyright 2020 IBM All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/hyperledger/fabric-sdk-go/pkg/core/config"
+	"github.com/hyperledger/fabric-sdk-go/pkg/gateway"
+)
+
+func main() {
+	fmt.Println("============ application-golang starts ============")
+
+	os.Setenv("DISCOVERY_AS_LOCALHOST", "true")
+	wallet, err := gateway.NewFileSystemWallet("wallet")
+	if err != nil {
+		fmt.Printf("failed to create wallet: %v\n", err)
+		os.Exit(1)
+	}
+
+	if !wallet.Exists("appUser") {
+		err = populateWallet(wallet)
+		if err != nil {
+			fmt.Printf("failed to populate wallet contents: %v\n", err)
+			os.Exit(1)
+		}
+	}
+
+	ccpPath := filepath.Join(
+		"..",
+		"..",
+		"test-network",
+		"organizations",
+		"peerOrganizations",
+		"org1.example.com",
+		"connection-org1.yaml",
+	)
+
+	gw, err := gateway.Connect(
+		gateway.WithConfig(config.FromFile(filepath.Clean(ccpPath))),
+		gateway.WithIdentity(wallet, "appUser"),
+	)
+	if err != nil {
+		fmt.Printf("failed to connect to gateway: %v\n", err)
+		os.Exit(1)
+	}
+	defer gw.Close()
+
+	network, err := gw.GetNetwork("mychannel")
+	if err != nil {
+		fmt.Printf("failed to get network: %v\n", err)
+		os.Exit(1)
+	}
+
+	contract := network.GetContract("basic")
+
+	result, err := contract.EvaluateTransaction("GetAllAssets")
+	if err != nil {
+		fmt.Printf("failed to evaluate transaction: %v\n", err)
+		os.Exit(1)
+	}
+	fmt.Println(string(result))
+
+	result, err = contract.SubmitTransaction("CreateAsset", "asset13", "yellow", "Tom", "5", "1300")
```

Comment:
The order of the parameters is wrong, you need to swap owner and size, Tom and 5.